docs/src/content/docs/book/message-mode.mdx

```diff
@@ -11,7 +13,7 @@ It's possible to use raw [`Int{:tact}`][int] values and manually provide them fo
 
 Mode value | Constant name                 | Description
 ---------: | :---------------------------- | -----------
-$0$        | -                             | Ordinary message (default).
+$0$        | `SendDefaultMode{:tact}`      | <Badge text="Since Tact 1.6" variant="tip"/> Ordinary message (default).
```

Comment:
```suggestion
$0$        | <Badge text="Since Tact 1.6" variant="tip"/> `SendDefaultMode{:tact}` | Ordinary message (default).
```

There are like 3 entries in this table, and won't be any more. So a bit of clutter on source side, while clearly displaying for the end-users is the way to go, I believe